src/it/resources/org/checkstyle/suppressionxpathfilter/parametername/SuppressionXpathRegressionParameterNameIgnoreOverridden.java

```diff
@@ -0,0 +1,9 @@
+package org.checkstyle.suppressionxpathfilter.parametername;
+
+class SuppressionXpathRegressionParameterNameIgnoreOverridden{
+  void method2(int V2) {} // warn
+  @Override
+  public boolean equals(Object V3) { // ok
+    return true;
+  }
+}
```

Comment:
Please add some line breaks in between, no need to compress so much, you can regularly format your code. Please do it for other files too.

Reply:
@0xbakry please separate methods and class definition by empty line. The only reason that we do not have a check on this for test input files is that sometimes we require poorly formatted input for test cases. If we do not have such a requirement for poorly formatted code for this test input, format code to make it easy to read.

Reply:
I got your point, thanks.
I updated it.

Reply:
Resolved.